packages/grafana-ui/src/components/Collapse/Collapse.tsx

```diff
@@ -110,6 +110,7 @@ export const ControlledCollapse: FunctionComponent<Props> = ({ isOpen, onToggle,
   return (
     <Collapse
       isOpen={open}
+      collapsible
```

Comment:
@hugohaggmark I made some changes here but not sure I understood fully how the props should be used. Here it seems like it does not make sense to have collapsible=false when this wrapper explicitly allows collapsing.

Reply:
Sorry, I don't remember this either @aocenas